docs/use-cases/exchange.md

```diff
@@ -0,0 +1,138 @@
+- [omg-js tutorials](https://docs.omg.network/network/start) - installation process, code samples, and various tips for using the omg-js library
+- [omg-js-samples](https://github.com/omgnetwork/omg-js-samples) - a JavaScript project that contains implementation for the core features of the OMG Network
+- [blockchain explorer](https://docs.omg.network/environments) - a tool that allows to view transactions and blocks that happen on the network
+
+#### 2.1.3 Make Deposits
+
+The `Deposit` terminology for an exchange has a different meaning from a deposit on the OMG Network. It's very important to distinguish between those two:
+1. OMG Network deposit - represents an ETH transaction to the corresponding `Vault` smart contract on the Ethereum network. This creates a new deposit on the OMG Network and allows the use of these funds until a user decides to withdraw them back to Ethereum.
+2. Exchange deposit - represents a standard transaction on the OMG Network. A safe number of confirmations for a given exchange should equal to a safe number of confirmations on the Ethereum network. This is known as [`deposit finality period`](https://docs.omg.network/glossary#deposit-finality-period) and is currently set to 10 blocks. The confirmations are counted on the Ethereum because the OMG Network relies on rootchain (Ethereum) security and creates blocks only when new transactions are being formed (i.e. on-demand). 
+
+While implementing a deposit functionality on a given exchange, you should refer to the latter definition. You can implement this step using [`Make a Transfer`](https://docs.omg.network/network/transfers) guide.
+
+Note, all of the code samples on the developer portal use Ropsten testnet as an example. The production version on the mainnet will have the same logic, except for the fees:
+- Ropsten - uses ETH as a fee
+- Mainnet - uses OMG token as a fee
+
+You can replace the `fee` object when constructing a transaction body with the OMG contract as follows:
+
+```
+fee: {
+  currency: O0xd26114cd6ee289accf82350c8d8487fedb8a0c07
+}
+```
+
+#### 2.1.4 Make Withdrawals
+
+Withdrawals on a given exchange also represent a standard transaction on the OMG Network. Therefore, you can use the step above.
+
+#### 2.1.5 Merge UTXO
+
+The OMG Network uses the UTXO-based model for keeping track of balances on its chain. Each transaction can have up to 4 UTXO as inputs and can create up to 4 UTXO as outputs. This means that depositing smaller amounts of funds will create UTXO of low value, and may create potential issues during funds withdrawal. Due to this, every exchange client should be merging UTXO more frequently, or even automated as a background service. This operation is free because it benefits the network.
+
+You can implement this functionality by following [Merging UTXO guide](https://docs.omg.network/network/utxos#merging-utxos). This is mostly due to 4 inputs limitation
+
+#### 2.1.6 Setup a Watcher
```

Comment:
maybe a paragraph or sentence(s) here that enforce reasons why exchanges **should** run their own Watcher